README.md

```diff
@@ -299,27 +298,44 @@ All the configurations are grouped in *sections* and some of them can vary depen
   - `ImageSize`: Configures the moedel input size. When the image has a different resolution, it is resized to fit the model ones. The available values of this parameter depends on the model chosen.
   - `ModelPath`: The same behavior as in the section `Detector`.
   - `MinScore`: Defines the facemask detection threshold. Any facemask detected by the model with a score less than the threshold will be ignored.
 
-
-- `[PostProcessor]`:
+- `[Tracker]`:
+  - `Name`: Name of the tracker used.
   - `MaxLost`: Defines the number of frames that an object should disappear to be considered as lost.
   - `TrackerIOUThreshold`: Configures the threshold of IoU to consider boxes at two frames as referring to the same object at IoU tracker.
-  - `NMSThreshold`: Configures the threshold of minimum IoU to detect two boxes as referring to the same object.
-  - `DefaultDistMethod`: Defines the default distance algorithm for the cameras without *DistMethod* configuration.
-  - `DistThreshold`: Configures the distance threshold for the *social distancing violations*
-  - `Anonymize`: A boolean parameter to enable/disable anonymization of faces in videos and screenshots.
-
-- `[Logger]`:
-  - `Name`: Configures the chosen logger. For now, we only support the *csv_logger*.
-  - `TimeInterval`: Sets the desired logging interval for objects detections and violations.
-  - `LogDirectory`: Defines the location where the generated files will be stored.
-  - `EnableReports`: A boolean parameter to enable/disable the reports generation.
-  - `HeatmapResolution`: Sets the resolution used by the heatmap report.
-  - `WebHooksEndpoint`: Configures an endpoint to receive in real-time the objects detections and violations.
+
+- `[SourcePostProcessor_N]`:
+
+  In the config files, we use the *SourcePostProcessor* sections to specifies additional processing steps in the sources. We support 3 different ones (identified by the field *Name*) that you enable/disable uncommenting/commenting them.
+    - `objects_filtering`: Used to remove invalid objects (duplicates or large).
+        - `NMSThreshold`: Configures the threshold of minimum IoU to detect two boxes as referring to the same object.
+    - `social_distance`: Used to measure the distance between objects and detect social distancing violations.
+      - `DefaultDistMethod`: Defines the default distance algorithm for the cameras without *DistMethod* configuration.
+      - `DistThreshold`: Configures the distance threshold for the *social distancing violations*
+    - `anonymizer`: A step used to enable anonymization of faces in videos and screenshots.
+
+- `[SourceLogger_N]`:
+
+  Similar to the section *SourcePostProcessor_N*, we support multiple loggers (right now 4) that you enable/disable uncommenting/commenting them.
+  - `video_logger`: Generates a video stream with the processing results. It is a useful logger to monitor in real-time your sources.
+  - `s3_logger`: Store the processed data in a S3 bucket.
```

Comment:
This currently stores only screenshots. Maybe add that here in the description. In future we could store all data, and then we would need more parameters

Reply:
Done